sources/Debate-frontend/src/app/services/debate.service.ts

```diff
@@ -3,16 +3,17 @@ import { Injectable } from '@angular/core';
 import { Observable } from 'rxjs';
 
 import { AddUpdateDebateDto } from '../dto/addUpdateDebate.dto';
+import { environment } from 'src/environments/environment';
 import { Debate } from '../dto/debate.dto';
 
 @Injectable({
   providedIn: 'root',
 })
 export class DebateService {
-  private readonly endpoint = 'http://localhost:8080/api/debate';
-  private readonly searchEndpoint = 'http://localhost:8080/api/search';
-  private readonly coverImageBaseUrl = 'http://localhost:8080/api/debate/cover/';
-  
+  private readonly endpoint = environment.api + '/debate';
+  private readonly searchEndpoint = environment.api + '/search';
+  private readonly coverImageBaseUrl = environment.api + '/debate.cover/';
```

Comment:
it is correct url?  Shouldn't look like /debate/cover/?

Reply:
```suggestion
  private readonly coverImageBaseUrl = environment.api + '/debate/cover/';
```